compiler/main/driver.cpp

```diff
@@ -1461,10 +1461,10 @@ static void postprocess_args() {
 
   postStackCheck();
 
-  postStaticLink();
-
   setMultiLocaleInterop();
 
+  postStaticLink();
```

Comment:
Why does this impact `CHPL_LIBTOOL_CMD`?

Reply:
It doesn't, the static flag doesn't have the proper behavior on `darwin` if that is not set. I suppose I can move it to another PR.

Reply:
I unswapped the calls.